DeepCat.js

```diff
@@ -202,17 +198,26 @@
 	}
 
 	function getSearchTerms( input ) {
-		return input.match( searchTermRegExp( keyString ) );
+		return input.match( '(-?\\b' + keyString + '\\s*'
+							+ '(?:("([^\\"]|\\.)+")' 			// search for keyword:"term including \"escaped\" quotes"
+							+ '|(?!' + keyString + ')\\S+))' 	// search for keyword:term excluding keyword term as parameter
```

Comment:
I'm not sure if we are talking about the same. Why do you need things that are not part of the deepcat search?
```js
'deepcat: a b'.match( '-?\\bdeepcat:\\s*(?:"(?:[^\\"]|\\.)+"|\\S+)' )
```
leads to `[ "deepcat: a" ]`.